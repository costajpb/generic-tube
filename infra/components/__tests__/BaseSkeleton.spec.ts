import snapshowWrapper from "@/infra/__tests__/snapshotWrapper"
import BaseSkeletonVue from "../BaseSkeleton.vue"

describe('infra/components/BaseSkeleton', () => {
    it('should render', () => {
        expect(snapshowWrapper(BaseSkeletonVue)).toMatchSnapshot()
    })
})