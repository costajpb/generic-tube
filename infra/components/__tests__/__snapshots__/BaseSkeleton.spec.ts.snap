// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`infra/components/BaseSkeleton > should render 1`] = `
<div
  class="_wrapper_587f33"
>
  
  
</div>
`;
